js/NewsView.js

```diff
@@ -0,0 +1,197 @@
+        }
+        let postElements = [{
+                id: `post${postNumber}`,
+                element: "div",
+                class: "flxRow allPosts",
+                parent: "content",
+                text: "",
+                event: "",
+                eventFunction: ""
+            },
+            {
+                id: `postDivisionLine1${postNumber}`,
+                element: "hr",
+                class: "divideLine",
+                parent: "content",
+                text: "",
+                event: "",
+                eventFunction: ""
+            },
+            {
+                id: `postImageId${postNumber}`,
+                element: "img",
+                class: "postImage",
+                parent: `post${postNumber}`,
+                text: "",
+                event: "",
+                eventFunction: ""
+            },
+            {
+                id: `postContent${postNumber}`,
+                element: "div",
+                class: "marginForPost",
+                parent: `post${postNumber}`,
+                text: "",
+                event: "",
+                eventFunction: ""
+            },
+            {
+                id: `postTitleId${postNumber}`,
+                element: "h2",
+                class: "marginForPost",
+                parent: `postContent${postNumber}`,
+                text: dataArray.postTitle,
+                event: "",
+                eventFunction: ""
+            },
+            {
+                id: `postTitleDescId${postNumber}`,
+                element: "h6",
+                class: "spacingWithInPosts",
+                parent: `postContent${postNumber}`,
+                text: `<span class=fontLight>Posted on </span>"  ${dataArray.postDate}  "<span class=fontLight>// Category :</span> "  ${dataArray.postCategory} "`,
+                event: "",
+                eventFunction: ""
+            },
+            {
+                id: `postDescId${postNumber}`,
+                element: "p",
+                class: "spacingWithInPosts",
+                parent: `postContent${postNumber}`,
+                text: dataArray.postDesc,
+                event: "",
+                eventFunction: ""
+            },
+            {
+                id: `postContinueBtnId${postNumber}`,
+                element: "button",
+                class: "postButton spacingWithInPosts",
+                parent: `postContent${postNumber}`,
+                text: "Continue Reading",
+                event: "",
+                eventFunction: ""
+            },
+        ];
+        let obj = new events();
+        this.createHTMLElement(postElements);
+        document.getElementById("postTitleDescId" + postNumber).value = dataArray.postCategory;
+        document.getElementById("postImageId" + postNumber).src = dataArray.imageSrc;
+        document.getElementById("postContinueBtnId" + postNumber).addEventListener("click",function(){obj.continueReading(this);});
```

Comment:
use call/bind/apply to pass instances instead of passing this directly a a parameter.